nin/backend/serve.js

```diff
@@ -92,8 +90,18 @@ var serve = function(projectPath, shouldRunHeadlessly) {
     var sockets = express();
     var sockets_server = require('http').createServer(sockets);
     var sock = socket(projectPath, function(conn) {
-      var sortedPaths = watcher.paths.sort();
+      var sortedPaths = watcher.paths.sort(function(a, b) {
+        var directoryPrecedence = {'lib': 0, 'src': 1, 'res': 2};
+        var directoryAScore = directoryPrecedence[a.slice(0, 3)];
+        var directoryBScore = directoryPrecedence[b.slice(0, 3)];
+        if(directoryAScore == directoryBScore) {
+          return a > b;
+        }
+        return directoryAScore > directoryBScore;
+
+      });
       for (var i in sortedPaths) {
+        console.log('first we need one of these!', sortedPaths[i]);
```

Comment:
This message seems a bit out of place. Maybe change the text or consider removing it. I guess it was for debug purposes?
![image](https://cloud.githubusercontent.com/assets/1488921/17623103/b7a13986-609e-11e6-8e81-dafabeacb1ad.png)